Dockerfile

```diff
@@ -3,6 +3,7 @@
 #     CGO_ENABLED=0 go build -a -tags netgo
 #     docker build --rm=true -t plugins/drone-cache .
 
-FROM gliderlabs/alpine:3.1
+FROM alpine:3.4
```

Comment:
Why not use `centurylink/ca-certs` as base image?

it reduces the overall image size.

https://github.com/appleboy/drone-facebook/blob/master/Dockerfile